config/environments/development.rb

```diff
@@ -9,7 +9,8 @@
   # it changes. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
   config.cache_classes = false
-
+  # TODO: Hadi - Need to discuss this...
+  config.active_storage.variant_processor = :mini_magick
```

Comment:
This is needed in order for the `representation' to work in rails.
We could discuss this further

Reply:
This is fine